### PR TITLE
[script.video.randomtv] 1.0.1

### PR DESCRIPTION
**Pull request by gmccauley**
### Description
RandomTV plays random TV episodes from your library. There are a few options: Include unwatched episodes (only watched will be included by default), Update Play Count, Repeat Playlist, Shuffle On Repeat, Show Notifications, Include All TV Shows or Select TV Shows

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0

### CODE REVIEW

File: script.video.randomtv/addon.py [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+	# Build Playlist
+	buildPlaylist(myEpisodes)
+
+	# Start Player
+	player.play(item=myPlaylist)
+	xbmc.sleep(100)
+#
+
+
+while (not xbmc.abortRequested):
```
Comment (enen92) [inline script.video.randomtv/addon.py]: this is deprecated I think. use xbmc.Monitor().abortrequested()

**Comment by gmccauley** [comment]
Sure is.  Thanks for the catch, I'll update it tonight.  I'm new to git and the wiki says only one commit per pull request, so I closed the last pull request and opened a new one.  Was that right or can I just add the commit to this request?

**Comment by Razzeee** [comment]
You have multiple ways, here are three from the top of my head:
- Just add more commits here, we can squash them for you, when we merge
- Try to amend your commit https://www.atlassian.com/git/tutorials/rewriting-history#git-commit--amend
- Commit and squash local then force push https://www.atlassian.com/git/tutorials/rewriting-history#git-rebase-i

**Comment by enen92** [comment]
Just add more commits to your branch and they will reflect on this PR. We squash them when merging. Its even better to review this way :)

**Comment by gmccauley** [comment]
Cool, thanks guys!

**Comment by enen92** [comment]
Awesome @gmccauley, thanks for adjusting

**Pull request merged**